hackathon_site/dashboard/frontend/src/components/cart/CartSummary/CartSummary.tsx

```diff
@@ -37,7 +39,12 @@ const CartSummary = () => {
                 color="primary"
                 variant="contained"
                 className={styles.btn}
-                disabled={cartQuantity === 0 || cartOrderLoading}
+                disabled={
+                    cartQuantity === 0 ||
+                    cartOrderLoading ||
+                    teamSize < 2 ||
```

Comment:
the team size constraints are constant but they may change based on what logistics decides, so instead of hardcoding the values here, can you make 2 constants in the src/constants.js file that have these constraints and reference them here.

Reply:
For some reason, this looks a bit ... messy (not ur fault tho, i would've done the same)
looking at the designs, I don't think we'll need to use the team size anywhere else so can you change your `teamSizeSelector ` to something like `teamMeetsSizeConstraintsSelector` that returns a boolean and do the min max logic there.